src/test/resources/ObjectCreationInsideAnonymousClass/expected/com/example/Simple.java

```diff
@@ -0,0 +1,18 @@
+package com.example;
+
+import org.testing.Baz;
+import org.testing.Foo;
+
+public class Simple {
+
+    public Baz getBaz() {
+        return new Baz() {
+
+            private Foo foo;
```

Comment:
The initializer of this field is part of the target method `getBaz()`, so I wouldn't expect it to be removed here.